flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -326,6 +320,8 @@ under the License.
 									<include>com.fasterxml.jackson.core:jackson-databind</include>
 									<include>com.fasterxml.jackson.core:jackson-annotations</include>
 									<include>commons-codec:commons-codec</include>
+									<include>org.apache.commons:commons-lang3</include>
```

Comment:
We don't need to include `commons-lang3`.

Reply:
Yes, thanks for the reminder.